opencga-app/app/scripts/azure/terraform/network-security.tf

```diff
@@ -0,0 +1,174 @@
+  source_address_prefix       = "*"
+  destination_address_prefix  = "*"
+  resource_group_name         = "${azurerm_resource_group.opencga.name}"
+  network_security_group_name = "${azurerm_network_security_group.nsg-web.name}"
+}
+
+resource "azurerm_network_security_rule" "allow-mongo-in" {
+  name                        = "allow-mongo-in"
+  priority                    = 3100
+  direction                   = "Inbound"
+  access                      = "Allow"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "27017"
+  source_address_prefix       = "*"
+  destination_address_prefix  = "*"
+  resource_group_name         = "${azurerm_resource_group.opencga.name}"
+  network_security_group_name = "${azurerm_network_security_group.nsg-mongo.name}"
+}
+
+resource "azurerm_network_security_rule" "allow-http-in-mongo" {
+  name                        = "allow-http-inbound-mongo"
+  priority                    = 3200
+  direction                   = "inbound"
+  access                      = "Allow"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "80"
+  source_address_prefix       = "*"
+  destination_address_prefix  = "*"
+  resource_group_name         = "${azurerm_resource_group.opencga.name}"
+  network_security_group_name = "${azurerm_network_security_group.nsg-mongo.name}"  
+}
+
+resource "azurerm_network_security_rule" "allow-ssh-in" {
+  name                        = "allow-ssh-in"
+  priority                    = 3200
+  direction                   = "Inbound"
+  access                      = "Allow"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "22"
+  source_address_prefix       = "*"
+  destination_address_prefix  = "*"
+  resource_group_name         = "${azurerm_resource_group.opencga.name}"
+  network_security_group_name = "${azurerm_network_security_group.nsg-daemonvm.name}"
+}
+
+resource "azurerm_network_security_rule" "allow-all-outbound-hdinsight" {
+  name                        = "allow-all-outbound-hdinsight"
+  priority                    = 3300
+  direction                   = "outbound"
+  access                      = "Allow"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "*"
+  source_address_prefix       = "*"
+  destination_address_prefix  = "*"
+  resource_group_name         = "${azurerm_resource_group.opencga.name}"
+  network_security_group_name = "${azurerm_network_security_group.nsg-hdinsight.name}"
+}
+
+resource "azurerm_network_security_rule" "allow-http-inbound-hdinsight" {
+  name                        = "allow-http-inbound-hdinsight"
+  priority                    = 3200
+  direction                   = "inbound"
+  access                      = "Allow"
+  protocol                    = "*"
+  source_port_range           = "*"
+  destination_port_range      = "80"
```

Comment:
Hi, this looks like it may not be the right ports for hdinsights https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-port-settings-for-services - why is port 80 open on this one?

Reply:
IIRC, the only reason for this was the certificate creation (SSL) for the cluster. At the moment, I left inbound 80 as allowed, until the certificate generation is changed to _internal_.

Reply:
This is on the `allow-http-inbound-hdinsight` resource though so think might be a copy paste issue. Or I'm confused.